<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateCustomer</class>
 <widget class="QDialog" name="updateCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>111</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Customer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 235, 255);</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="telLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Tel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="discountLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>35</width>
     <height>10</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>31</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 140, 255);
font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>57</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 140, 255);
font: 10pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="telLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="discountLineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="nameComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rejectButton</sender>
   <signal>clicked()</signal>
   <receiver>updateCustomer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>updateCustomer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nameComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>updateCustomer</receiver>
   <slot>refresh_line_edits(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refresh_line_edits(QString)</slot>
 </slots>
</ui>
